simulator-ui/src/main/resources/static/tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "typeRoots": [
       "node_modules/@types"
     ],
+    "skipLibCheck": true,
```

Comment:
that does what exactly?

Reply:
When a program includes large declaration files, the compiler spends a lot of time type checking declarations that are already known to not contain errors, and compile times may be significantly shortened by skipping declaration file type checks.

Reply:
In addition, it throws the following error if I omit  ""skipLibCheck": true":
![image](https://user-images.githubusercontent.com/92145959/144383739-8e8b36b7-62ca-4530-86e6-40a5a38f8cbd.png)

Reply:
removed/solved!